app/src/main/java/com/nextcloud/talk/ui/dialog/SetStatusDialogFragment.kt

```diff
@@ -244,6 +283,42 @@ class SetStatusDialogFragment :
         }
     }
 
+    override fun revertStatus() {
+        if (isRestoreStatusAvailable(currentUser!!)) {
+            ncApi.revertStatus(
+                credentials,
+                ApiUtils.getUrlForRevertStatus(currentUser?.baseUrl!!, currentStatus?.messageId)
+            )
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(object : Observer<GenericOverall> {
+
+                    override fun onSubscribe(d: Disposable) {
```

Comment:
we should add disposable handling if we still continue to add RxJava stuff (it was also not implemented in the past too often).
Could you add to dispose disposables, also for the other api requests in this fragment? Just take a look at other classes, e.g. AccountVerificationActivity how it's done.